pydatalab/pydatalab/routes/v0_1/roles.py

```diff
@@ -0,0 +1,58 @@
+from bson import ObjectId
+from flask import Blueprint, jsonify, request
+from flask_login import current_user
+
+from pydatalab.config import CONFIG
+from pydatalab.mongo import flask_mongo
+
+role = Blueprint("roles", __name__)
+
+
+@role.route("/roles/<user_id>", methods=["PATCH"])
+def save_role(user_id):
+    request_json = request.get_json()
```

Comment:
This looks fine for now; we might consider merging it with the general `save_user` update route but perhaps this is cleaner as an admin-only thing for now